src/utils/components/Charts/NetworkUtil/NetworkThresholdChartSingleSource.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import xbytes from 'xbytes';
+
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { PrometheusValue } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Chart,
+  ChartArea,
+  ChartAxis,
+  ChartGroup,
+  ChartVoronoiContainer,
+} from '@patternfly/react-charts';
+import chart_color_blue_400 from '@patternfly/react-tokens/dist/esm/chart_color_blue_300';
+import useDuration from '@virtualmachines/details/tabs/metrics/hooks/useDuration';
+
+import ComponentReady from '../ComponentReady/ComponentReady';
+import useResponsiveCharts from '../hooks/useResponsiveCharts';
+import { tickFormat } from '../utils/utils';
+
+type NetworkThresholdSingleSourceChartProps = {
+  data: PrometheusValue[];
+  link: string;
+};
+
+const NetworkThresholdSingleSourceChart: React.FC<NetworkThresholdSingleSourceChartProps> = ({
+  data,
+  link,
+}) => {
+  const { currentTime, duration } = useDuration();
+
+  const { ref, width, height } = useResponsiveCharts();
+
+  const chartData = data?.map(([, item], index) => {
+    return { x: index, y: +item, name: 'Network In' };
+  });
+
+  const isReady = !isEmpty(chartData);
+
+  return (
+    <ComponentReady isReady={isReady}>
+      <div className="util-threshold-chart" ref={ref}>
+        <Link to={link}>
+          <Chart
+            height={height}
+            width={width}
+            padding={35}
+            scale={{ x: 'time', y: 'linear' }}
+            containerComponent={
+              <ChartVoronoiContainer
+                labels={({ datum }) => {
+                  return `${datum?.name}: ${xbytes(datum?.y, { iec: true, fixed: 2 })}`;
+                }}
+                constrainToVisibleArea
+              />
+            }
+          >
+            <ChartAxis
+              tickFormat={tickFormat(duration, currentTime)}
+              style={{
+                ticks: { stroke: 'transparent' },
+              }}
+              axisComponent={<></>}
```

Comment:
?

Reply:
In order to override default component as it not needed